packages/frontend/electron/src/helper/db/base-db-adapter.ts

```diff
@@ -29,6 +29,7 @@ export abstract class BaseSQLiteAdapter {
       if (maxVersion !== WorkspaceVersion.Surface) {
         await migrateToLatest(this.path, WorkspaceVersion.Surface);
       }
+      await applyGuidCompatibilityFix(this.db);
```

Comment:
can we move this process as a migrate process?

Reply:
once migrated，never pain

Reply:
DB operations in electron is already in a help process. I guess what we shall do is to migrate in the browser